uitools/import/Esri/ArcGISRuntime/Toolkit/SearchView.qml

```diff
@@ -327,12 +327,19 @@ Pane {
             }
             // Simple highlight rectangle for selected result.
             highlightFollowsCurrentItem: false
-            highlight: Rectangle {
+            highlight: Pane {
                 x: list.x
                 width: list.width
                 y: list.currentItem ? list.currentItem.y : 0
                 height: list.currentItem ? list.currentItem.height : 0
-                color: searchView.palette.highlight
+                background: Rectangle {
+                    color: searchView.palette.highlight
+                }
+                leftPadding: 0
+                rightPadding: 0
+                topPadding: 0
+                bottomPadding: 0
+
```

Comment:
❌  I'd leave this as a rectangle if we are customizing the color.

💭  The real problem I believe is we're using the wrong kind of Delegate, we should be using an `ItemDelegate` and doing the highlighting via that's inbuilt highlighting methods, see the BasemapGallery for example. This is not a request to change the rendering of SearchView, more just a comment of what the "proper" fix should be.